Core/Helper.php

```diff
@@ -290,4 +299,26 @@ public static function isThisModule($sModuleId)
     {
         return $sModuleId === self::MODULE_ID;
     }
+
+    /**
+     * @return array
+     *
+     * @since 1.0.0
+     */
+    public static function getShopInfoFields()
+    {
+        $shopId = Registry::getConfig()->getShopId();
```

Comment:
Rename variable names according to Oxid standards ($shopId -> $sShopId ...)

Reply:
👍